evaluator/planning_evaluator/src/metrics/deviation_metrics.cpp

```diff
@@ -77,5 +77,26 @@ Stat<double> calcVelocityDeviation(const Trajectory & ref, const Trajectory & tr
   return stat;
 }
 
+Stat<double> calcPoseLongitudinalDeviation(const Pose & base_pose, const Pose & target_pose)
```

Comment:
`const Pose & target_pose`
can be replaced with 
`const Point & target_point`?
if direction of target_pose is not used.

Reply:
thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/3053/commits/31c8b82f24c5d381c9bf12581bdfeb6f8ff4cf50

Reply:
(and https://github.com/autowarefoundation/autoware.universe/pull/3053/commits/4ac91a03e16fd568295cb92b3f981dba27e5e7fc)